src/main/resources/db/migration/V202409081401__create_pet.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE pets
```

Comment:
By the way, it's better to be consistent, and the name should be 'pet' in the singular